modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditApp.es.js

```diff
@@ -178,7 +178,7 @@ export default ({
 												'there-are-no-table-views-yet'
```

Comment:
Good point @rafaprax, i didn't pay attention to that, because has not changes on this PR.
For this case we don't have an empty state for no content found, because we always display the "No Workflow" option. So I think that we can remove this messages if the component allows it. Makes sense? @gabrielpramos @naraandrade

Reply:
yeah, I've already removed, good point you guys ;D

Reply:
Yes, these language keys are being passed because the empty state is mandatory in this component (for now), but they have no impact on the final result.

Reply:
the high order function "withEmpty" takes the emptyState as undefined if you guarantee the items array has at least the "No Workflow" item